lib/main.js

```diff
@@ -394,7 +394,7 @@ define(['./errors'], function(errors) {
     // for easy membership testing with the "in" keyword
     function setLike(list, fill /* default true*/) {
         if(fill === undefined) fill = true;
-        var setLike = {};
+        var setLike = Object.create(null);
```

Comment:
this is considered a JS anti-parttern, with the previous code preferred - why the switch? (always interested in technical details)